orchestra/todos/urls.py

```diff
@@ -29,4 +29,7 @@
     url(r'^worker_task_recent_todo_qas/$',
         views.worker_task_recent_todo_qas,
         name='worker_task_recent_todo_qas'),
+    url(r'^import_todo_list_template_from_spreadsheet/(?P<pk>[0-9]+)/$',
```

Comment:
feels a bit more RESTful to structure this as something like `todolist_templates/<pk>/import`, no?

Reply:
done!